flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -370,7 +374,7 @@ private void testKeyedListStateUpgrade(
     //  Tests for keyed MapState
     // -------------------------------------------------------------------------------
 
-    @Test
+    @TestTemplate
     public void testKeyedMapStateAsIs() throws Exception {
```

Comment:
```suggestion
    void testKeyedMapStateAsIs() throws Exception {
```
could be package private